### ISSUE TITLE
Module parse failed: Unexpected token

### ISSUE THREAD

**Issue by DmitryEfimenko** [hankchiutw/subjectize#2]
I'm getting an error when building an app that uses this package in the --prod mode using target es6.
```
ERROR in ./node_modules/subjectize/index.js 1:99
Module parse failed: Unexpected token (1:99)
File was processed with these loaders:
 * ./node_modules/@angular-devkit/build-optimizer/src/build-optimizer/webpack-loader.js
You may need an additional loader to handle the result of these loaders.
> function n(s,r,i={}){let t=Symbol(r),e=Object.getOwnPropertyDescriptor(s,r),{get:c,set:u}=i,f=c||e?.get||function(){return e?.value&&this[t]===void 0&&(this[t]=e.value),this[t]};Object.defineProperty(s,r,{get:f,set:function(o){this[t]=o,e?.set&&e.set.call(this,o),u?.call(this,o)},configurable:!0})}function x(s){return(r,i)=>{n(r,i,{set(t){this[s]!==void 0&&t.next(this[s])}}),n(r,s,{set:function(t){this[i]!==void 0&&this[i].next(t)}})}}function d(s){return(r,i)=>{n(r,i,{set(t){s.forEach(e=>{this[e]!==void 0&&t.next([e,this[e]])})}}),s.forEach(t=>{n(r,t,{set:function(e){this[i]!==void 0&&this[i].next([t,e])}})})}}export{x as Subjectize,d as SubjectizeProps};
|
```

**Comment by hankchiutw** [comment]
Related to https://github.com/snowpackjs/snowpack/discussions/2354

**Comment by hankchiutw** [comment]
Fixed by 621853e0025f4650710a5a184d3d23f8282b169a

**Comment by DmitryEfimenko** [comment]
Confirming that this fixed the issue. Thanks for the quick turnaround!
